chore(release): update package versions

abbeyhrt opened the pull request:
Changes:
 - @carbon/cli: 10.5.0 => 10.6.0-rc.0
 - @carbon/colors: 10.5.0 => 10.6.0-rc.0
 - carbon-components: 10.7.0 => 10.8.0-rc.0
 - @carbon/elements: 10.7.0 => 10.8.0-rc.0
 - @carbon/grid: 10.6.0 => 10.7.0-rc.0
 - @carbon/icons-handlebars: 10.6.0 => 10.7.0-rc.0
 - @carbon/icons-react: 10.6.0 => 10.7.0-rc.0
 - @carbon/icons-vue: 10.6.0 => 10.7.0-rc.0
 - @carbon/icons: 10.6.0 => 10.7.0-rc.0
 - @carbon/layout: 10.5.0 => 10.6.0-rc.0
 - carbon-components-react: 7.7.0 => 7.8.0-rc.0
 - @carbon/themes: 10.7.0 => 10.8.0-rc.0
 - @carbon/type: 10.6.0 => 10.7.0-rc.0

Reviewers:

Double check that packages were updated to the correct versions
Double check that all tests are passing ✅
Double check that everything is working as expected

abbeyhrt committed:
chore(release): update package versions

abbeyhrt committed:
chore(release): update carbon/cli

alisonjoseph commented:
LGTM once conflict is fixed

abbeyhrt committed:
Merge branch 'master' into chore/release-v10.8.0-rc.0

vpicone commented on packages/themes/package.json:
```diff
@@ -24,14 +24,13 @@
   "dependencies": {
-    "@carbon/colors": "10.5.0",
-    "@carbon/type": "10.6.0",
-    "@carbon/layout": "10.5.0"
```
Did we loose a dependency here?

abbeyhrt replied:
Yeah, I don't know why. I'll add it back in!

abbeyhrt committed:
chore(release): add in dropped dependency

abbeyhrt committed:
Merge branch 'chore/release-v10.8.0-rc.0' of https://github.com/abbeyhrt/my-carbon-fork into chore/release-v10.8.0-rc.0

abbeyhrt committed:
Merge branch 'master' into chore/release-v10.8.0-rc.0

abbeyhrt committed:
chore(release): sync generated docs

abbeyhrt committed:
Merge branch 'chore/release-v10.8.0-rc.0' of https://github.com/abbeyhrt/my-carbon-fork into chore/release-v10.8.0-rc.0

abbeyhrt committed:
chore(release): sync generated files

asudoh committed:
chore(tabs): rename fixed variant to container (#4685)

Fixes #4683.

abbeyhrt committed:
chore(release): update package dependencies

abbeyhrt committed:
chore(release): update package dependencies

abbeyhrt committed:
Revert "chore(release): update package dependencies"

This reverts commit 785bdcdc3d2637c384b7f4a2099fa70d3c9be896.

abbeyhrt committed:
chore(release): updates pacakge dependencies

chrisconnors-ibm committed:
feat(icons): Watson discovery icons (#4696)

adds:
improve-relevance.svg
phrase-sentiment.svg
folders.svg
exit.svg
document-sentiment.svg
character-patterns.svg

updates categories.yml and metadata.yml

joshblack committed:
chore(package): update package versions

joshblack committed:
chore(project): sync generated files

joshblack commented:
@abbeyhrt gonna close this to pursue 10.9 like we were talking about!
